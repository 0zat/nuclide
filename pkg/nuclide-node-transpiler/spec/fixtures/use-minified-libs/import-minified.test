import Rx from 'rxjs';
import {Observable} from 'rxjs';

assert(Rx.Observable === Observable);
