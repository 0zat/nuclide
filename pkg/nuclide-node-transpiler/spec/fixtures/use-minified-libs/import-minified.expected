function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

var _rxjsBundlesRxMinJs = require('rxjs/bundles/Rx.min.js');

var _rxjsBundlesRxMinJs2 = _interopRequireDefault(_rxjsBundlesRxMinJs);

assert(_rxjsBundlesRxMinJs2['default'].Observable === _rxjsBundlesRxMinJs.Observable);
